packages/lesswrong/components/hooks/usePostsListView.tsx

```diff
@@ -16,8 +17,11 @@ type PostsListViewContext = {
   setView: (view: PostsListViewType) => void,
 }
 
+const defaultViewSetting = postsListViewTypeSetting.get()
+const defaultView: PostsListViewType = isPostsListViewType(defaultViewSetting) ? defaultViewSetting : "list"
```

Comment:
This should probably be wrapped in a function, since otherwise it assigns whatever value is in the database to the `defaultView` for the lifetime of the server it's running on, rather than getting whatever the latest value is for that db setting (which is updated every 5 minutes)